pkg/placement/membership.go

```diff
@@ -224,6 +226,10 @@ func (p *Service) processRaftStateCommand(stopCh chan struct{}) {
 		case <-p.shutdownCh:
 			return
 
+		case <-processingTicker.C:
+			log.Infof("process raft state command: nothing happened...")
+			continue
```

Comment:
The logging print is like a heartbeat that prints to console to indicate the node is alive.

Reply:
This should probably be a debug log?